Framework/Source/Utils/RenderGraphUI.cpp

```diff
@@ -409,16 +411,19 @@ namespace Falcor
         std::experimental::filesystem::last_write_time(filePath, std::chrono::system_clock::now());
     }
 
-    RenderGraphUI::RenderGraphUI(RenderGraph& renderGraphRef)
-        : mRenderGraphRef(renderGraphRef), mNewNodeStartPosition(-40.0f, 100.0f)
+    RenderGraphUI::RenderGraphUI(RenderGraph& renderGraphRef, const std::string& renderGraphName)
```

Comment:
You rely on a lot of static variables, make sure that still works if someone creates multiple RenderGraphUIs (or make it a singleton)